site/en/docs/privacy-sandbox/attribution-reporting-event-introduction/index.md

```diff
@@ -0,0 +1,321 @@
+ 
+You can change this behavior by configuring [source-side priorities](/docs/privacy-sandbox/attribution-reporting-event-guide/#define-custom-attribution-rules).
+ 
+#### Example
+ 
+A user visits `news.example` and clicks an ad for `shoes.example`. On the next day, they visit `other-news.example` and again click an ad for `shoes.example`. A few days later, they visit `shoes.example` and convert by making a purchase.
+
+By default, unless you configure custom [source-side priorities](/docs/privacy-sandbox/attribution-reporting-event-guide/#define-custom-attribution-rules), you'll receive a report that attributes this conversion to the ad clicked on `other-news.example`.
+
+### Matching source and trigger data
+
+Let's take a given ad click (source event) with an click ID (source event ID) of `1298765678762`.
+When this click is registered, the browser⏤on the user's device⏤stores the click ID `1298765678762`, and alongside it:
+
+- `attributiondestination`. Typically, this is an advertiser site. Example: `shoes.example`.
+- `attributionreportto`. Typically, this is an adtech endpoint. Example: `adtech.example`.
+
+Later on, a trigger (conversion) takes place on a page.
+The browser will match this trigger with the source event ID `1298765678762` in the following conditions:
+
+- The page on which the trigger (conversion) took place matches **the eTLD+1 that was provided in the source's `attributiondestination`**. Example: `shoes.example/checkout`.
+- The attribution is triggered (that is, the conversion is registered) via a request to the **same `attributionreportto` origin** (that then redirects to a `.well-known` location). Example: `adtech.example`.
+
+<figure>
+{% Img src="image/O2RNUyVSLubjvENAT3e7JSdqSOx1/gdpjvYVB50vCbBUP2UKj.png", alt="ALT_TEXT_HERE", width="800", height="474" %}
+ <figcaption>How the browser matches click IDs (source event IDs) with attribution trigger data (conversion data).</figcaption>
+</figure>
+ 
+Note:
+ 
+- When a source is registered with a given (`attributiondestination`, `attributionreportto`) pair, existing sources matching this pair will be looked up in storage. If the matching sources have been triggered at least once (that is, have scheduled a report), they will be removed from browser storage and will not be eligible for further reporting. Any pending reports for these sources will still be sent.
+- For click events, a source will be registered only if the resulting document being navigated to ends up sharing the eTLD+1 with the `attributiondestination` origin.
+
+### User-initiated data clearing
+
+Data clearing on the user's end⏤for example, when a user deletes their browsing history⏤can affect measurement based on the Attribution Reporting API in ways that differ from cookie-based measurement.
+
+Head over to [FAQ: Impact of user-initiated data clearing on attribution reports](/docs/privacy-sandbox/attribution-reporting-data-clearing/) for details.
+
+## Summary: Browser API parameters
```

Comment:
Hm, not all of these are API parameters. Any ideas for a more appropriate subheading?

Reply:
Oh right, I went for 
`## Summary: API behavior and parameters`
LMK if this doesn't work!